docs/user_guide/emummc/entering_rcm.md

```diff
@@ -7,41 +7,43 @@ There are several methods of entering RCM (**R**e**C**overy **M**ode). The most
 &nbsp;
 
 !!! danger "Patched Switch"
-    Note that patched units **can** enter RCM, but it is not possible to send a payload on those systems. Also note that RCM is a different recovery mode than the one accessed by holding Volume Up, Volume Down, and Power.
+    Note that all patched units **can** enter RCM, but it is not possible to send a payload on those systems. Also note that RCM is a different recovery mode than the one accessed by holding Volume Up, Volume Down, and Power (usually referred to as Maintenance Mode).
 
 !!! snippet "Note"
-    The order of methods on this page is in the order of ease. The easiest to immediately accomplish are listed at the top, and the most advanced/difficult methods are at the bottom.
+    The order of methods on this page is in the order of convenience. The easiest to immediately accomplish are listed at the top, and the most advanced/difficult methods are at the bottom.
 
 &nbsp;
 
 ### Instructions
 
 !!! tip ""
-    1. Power off the Switch and use one of the methods listed below to short the pins on the right joycon rail.
-    2. Hold Volume Up and press the Power button.
+    1. Unplug any USB-C cables from the console.
+    2. Power off the Switch and use one of the methods listed below to short the pins on the right Joy-Con rail.
+    3. Hold Volume Up and press the Power button.
 
-    If your Switch displays the Nintendo logo and boots normally or immediately shuts down, you didn't successfully enter RCM and should try again. Otherwise, if your console did not turn on normally, and the screen remained black with no backlight, your Switch is in RCM.
+    Your console should appear to have not responded, frozen on a black screen. *This is expected behavior.*. If you see the Nintendo > Nintendo Switch logo, then your Switch has booted up normally, and you'll need to attempt again.
 
 !!! tip ""
     ### RCM Jig (Easiest for beginners)
     <details>
         <summary>Click to expand!</summary>
 
     !!! snippet "Note"
-        Some jig designs use paperclips, inheriting the same risks as the Metal Bridge / Paperclip method.
+        Some jig designs use paperclips, inheriting the [same risks as using a paperclip outright](../faq/#what-makes-a-good-jig-good-can-i-use-a-paperclip).  
+        Please do not suggest to (or attempt to) use a paperclip or other foreign object to enter RCM.
```

Comment:
"Please do not attempt to" should be enough, the guide is targeting helpees, not helpers

Reply:
I don't think there's anything wrong with that wording.